config/routes.rb

```diff
@@ -0,0 +1,17 @@
+Rails.application.routes.draw do
+  root 'trips#home'
+  get 'trips/home', to: 'trips#home', as: 'trips_home'
+  resources :trips
+
+  resources :passengers do
+  get '/trips/:id/in_progress', to: 'trips#in_progress', as: 'trip_inprogress'
+    resources :trips, only:  [:create, :show]
```

Comment:
Do you need the` show `path for a trip in this context?  With `show` You have the trip id, so you don't need the passenger_id. 

You also shouldn't have this indented under `get '/trips/:id/in_progress', to: 'trips#in_progress', as: 'trip_inprogress'`